File: app/src/main/java/n7/ad2/ui/setting/SettingActivity.java
package n7.ad2.ui.setting;

import android.os.Bundle;
import android.view.MenuItem;

import androidx.databinding.DataBindingUtil;

import n7.ad2.R;
import n7.ad2.databinding.ActivitySettingBinding;
import n7.ad2.ui.setting.SettingsFragment;
import n7.ad2.utils.BaseActivity;

public class SettingActivity extends BaseActivity {

    private ActivitySettingBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = DataBindingUtil.setContentView(this, R.layout.activity_setting);

            getSupportFragmentManager()
                    .beginTransaction()
                    .replace(R.id.container_activity_setting, new SettingsFragment())
                    .commit();


        setToolbar();
    }

    private void setToolbar() {
        setSupportActionBar(binding.toolbarActivitySetting);
        binding.toolbarActivitySetting.setTitle(R.string.setting);
        if (getSupportActionBar() != null) {
            getSupportActionBar().setDisplayShowHomeEnabled(true);//включаем кнопку домой
            getSupportActionBar().setDisplayHomeAsUpEnabled(true);//ставим значок стрелочки на кнопку
        }
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                onBackPressed();
                break;
        }
        return super.onOptionsItemSelected(item);
    }

}

File: app/src/main/java/n7/ad2/ui/setting/SettingActivity.kt
package n7.ad2.ui.setting

import android.os.Bundle
import android.view.MenuItem
import androidx.databinding.DataBindingUtil
import androidx.fragment.app.commit
import n7.ad2.R
import n7.ad2.databinding.ActivitySettingBinding
import n7.ad2.utils.BaseActivity

class SettingActivity : BaseActivity() {

    private lateinit var binding: ActivitySettingBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.setContentView(this, R.layout.activity_setting)
        supportFragmentManager.commit {
            replace(binding.container.id, SettingsFragment())
        }
        setToolbar()
    }

    private fun setToolbar() {
        setSupportActionBar(binding.toolbarActivitySetting)
        supportActionBar?.let {
            it.setTitle(R.string.setting)
            it.setDisplayShowHomeEnabled(true) //включаем кнопку домой
            it.setDisplayHomeAsUpEnabled(true) //ставим значок стрелочки на кнопку
        }
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) onBackPressed()
        return super.onOptionsItemSelected(item)
    }
}